File: Task/99_Bottles_of_Beer/JavaScript/99_bottles_of_beer.js
// Line breaks are in HTML
var beer = 99;
while (beer > 0)
{
 document.write( beer + " bottles of beer on the wall<br>" );
 document.write( beer + " bottles of beer<br>" );
 document.write( "Take one down, pass it around<br>" );
 document.write( (beer - 1) + " bottles of beer on the wall<br>" );
 beer--;
}

File: Task/99-bottles-of-beer/F-Sharp/99-bottles-of-beer.fs
#light
let rec bottles n =
    let (before, after) = match n with
                          | 1 -> ("bottle", "bottles")
                          | 2 -> ("bottles", "bottle")
                          | n -> ("bottles", "bottles")
    printfn "%d %s of beer on the wall" n before
    printfn "%d %s of beer" n before
    printfn "Take one down, pass it around"
    printfn "%d %s of beer on the wall\n" (n - 1) after
    if n > 1 then
        bottles (n - 1)
